src/components/Notifications/Form/IntegrationRecipientTypeahead.tsx

```diff
@@ -102,6 +114,7 @@ export const IntegrationRecipientTypeahead: React.FunctionComponent<IntegrationR
                 onFilter={ onFilter }
                 menuAppendTo={ document.body }
                 isDisabled={ props.isDisabled }
+                validated='default'
```

Comment:
This could change to 
```suggestion
                validated={ props.error ? 'error' : undefined  }
```